packages/client/modules/userDashboard/components/OrgBilling/OrgPlanDrawer.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import {commitLocalUpdate, useFragment} from 'react-relay'
+import {OrgPlanDrawer_organization$key} from '../../../../__generated__/OrgPlanDrawer_organization.graphql'
+import ResponsiveDashSidebar from '../../../../components/ResponsiveDashSidebar'
+import {Close} from '@mui/icons-material'
+import {PALETTE} from '../../../../styles/paletteV3'
+import {BezierCurve, Breakpoint, DiscussionThreadEnum, ZIndex} from '../../../../types/constEnums'
+import LabelHeading from '../../../../components/LabelHeading/LabelHeading'
+import {desktopSidebarShadow} from '../../../../styles/elevation'
+import useBreakpoint from '../../../../hooks/useBreakpoint'
+import PlainButton from '../../../../components/PlainButton/PlainButton'
+import useAtmosphere from '../../../../hooks/useAtmosphere'
+import {ICON_SIZE} from '../../../../styles/typographyV2'
+import OrgPlanDrawerContent from './OrgPlanDrawerContent'
+
+const DrawerHeader = styled('div')({
+  alignItems: 'center',
+  display: 'flex',
+  justifyContent: 'space-between',
+  padding: '16px 8px 16px 16px',
+  width: '100%'
+})
+
+const Drawer = styled('div')<{isDesktop: boolean; isOpen: boolean}>(({isDesktop, isOpen}) => ({
+  boxShadow: isDesktop ? desktopSidebarShadow : undefined,
+  backgroundColor: '#FFFFFF',
+  display: 'flex',
+  flex: 1,
+  flexDirection: 'column',
+  justifyContent: 'stretch',
+  overflow: 'hidden',
+  position: isDesktop ? 'fixed' : 'static',
+  bottom: 0,
+  top: 0,
+  right: isDesktop ? 0 : undefined,
+  transition: `all 200ms ${BezierCurve.DECELERATE}`,
+  userSelect: 'none',
+  transform: `translateX(${isOpen ? 0 : DiscussionThreadEnum.WIDTH}px)`,
+  width: DiscussionThreadEnum.WIDTH,
+  zIndex: ZIndex.SIDEBAR,
+  height: '100%',
+  '@supports (height: 1svh) and (height: 1lvh)': {
+    height: isDesktop ? '100lvh' : '100svh'
+  }
+}))
+
+const CloseIcon = styled(Close)({
+  color: PALETTE.SLATE_600,
+  cursor: 'pointer',
+  '&:hover': {
+    opacity: 0.5
```

Comment:
We seem to have this close button style in quite a few other places (~7 in total), e.g. https://github.com/ParabolInc/parabol/blob/c7b72ac1a722c4f155af8f6cdf49d1042ad3fefd/packages/client/components/TeamPrompt/TeamPromptDiscussionDrawer.tsx#L64-L66. 

I think the style in your gif looks good, though. I imagine it's using a `FlatButton` instead of a `PlainButton`. Do you know what component it's from? I'd be happy to change the style here as long as it's consistent with the rest of the app